src/extension.ts

```diff
@@ -327,12 +327,22 @@ export class Extension implements RunHooks {
     this._workspaceObserver.setWatchFolders(this._models.testDirs());
   }
 
-  private _envProvider(): NodeJS.ProcessEnv {
+  private _envProvider: EnvProvider = ({ workspaceFolder }) => {
+    this._vscode.workspace.getWorkspaceFolder(this._context.extensionUri);
```

Comment:
todo rm.
I can also get workpaceFolder from `this._vscode.workspace.rootPath` but not sure if it's correct for multiworkspace editors